lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -272,5 +274,9 @@ def initialize_from_hash(hash)
         end
       end
     end
+
+    def default_port
```

Comment:
Why not have an `attr_reader :default_port` and set it in `DatabaseConfiguration#initialize`